### PR TITLE
now everything is saved and changed

### LINKED ISSUE #71: Make create_new_test/user
Make create_new_test/user funcs in https://github.com/IlyaM1/TestBuilder/blob/main/Interface/Admin_view_tests_and_users.py
That funcs creates new test/user in db

### LINKED ISSUE #76: Save test/user in Admin_test/user_view.py
After editing test or user by admin and pushing save button - make saving test/user in DB

### PR DESCRIPTION
**Pull request by akrisfx**
the function of adding and update a new/old test/user to the database has been added
- in Admin_user/test_view.py added add_new_test/user functionality
- and in Admin_view_tests_and_users.py
- fixed sql_get_user_with_id
resolved #71
resolved #76

### CODE REVIEW

File: Interface/Admin_user_view.py [hunk 1]
```diff
@@ -158,15 +160,21 @@ def save_user_button_pushed(self):
...
 
         cfg = Config()
         user_db = SQLInteract(table_name='testcase', filename_db=cfg.config["path"] + '/db/users.db')
-        user_db.sql_update_one_by_id(update_field="name", update_value="" + input_name + "", search_id=self.user["id"])
-        user_db.sql_update_one_by_id(update_field="password", update_value=input_password, search_id=self.user["id"])
-        user_db.sql_update_one_by_id(update_field="post", update_value=input_post, search_id=self.user["id"])
+        if self.is_new_user:
+            user_db.sql_add_new_user(self.user)
+        else:
+            user_db.sql_update_one_by_id(update_field="name", update_value="" + input_name + "", search_id=self.user["id"])
```
Comment (IlyaM1) [inline Interface/Admin_user_view.py:172]: "" + input_name + "" - ???

Comment (IlyaM1) [inline Interface/Admin_user_view.py:172]: update_value="" + input_name + ""
???

**Review by IlyaM1 — APPROVED** [review]
Fine

**Comment by akrisfx** [comment]
> Fine

А чо не "норм"

**Pull request merged**